examples/snippets/antora.yml

```diff
@@ -0,0 +1,3 @@
+name: "docsnippets"
+version: 1.3.1
```

Comment:
Do we need the version hardcoded here?
Every additional place that hardcodes our version number is one more problem to solve for automated release management.

Reply:
Yes, we will need a specific version number, to support multi-version documentation
On master, it can be `latest` on something like that. But in the branches, it should be the actual version.

Reply:
How can we keep this maintainable? 
Should this be auto-updated by the release automation?